File: Hacker_rank/Strings/lover_letter.cpp
#include <iostream>

using namespace std;

int main()
{
  int t, left, right, ans;
  string str;
  
  scanf("%d",&t);

  while(t--)
  {
    cin >> str;
    
    left = ans =  0;
    right = str.length() - 1;

    while(left <= right) 
      ans += abs( str[left++] - str[right--] );

    printf("%d\n", ans); 
  }
  
  return 0;
}

File: Hacker_rank/Strings/lover_letter.rb
t = gets.chomp.to_i

t.times do
  str = gets.chomp
  left = ans = 0
  right = str.length - 1

  while left <= right do
    ans += ( str[left].ord - str[right].ord ).abs
    left += 1
    right -= 1
  end

  puts ans
end
